xonsh/commands_cache.py

```diff
@@ -274,7 +274,7 @@ def default_predictor_alias(self, cmd0):
         alss = getattr(builtins, "aliases", dict())
         while cmd0 in alss:
             alias_name = alss[cmd0]
-            if not isinstance(alias_name, list):
+            if not isinstance(alias_name, cabc.Sequence):
```

Comment:
should this be `if isinstance(alias_name, (str, bytes)) or not isinstance(alias_name, cabc.Sequence)`?

Reply:
Yes, I were right. I changed my commit.